tests/ha/test_data_pod_failure.py

```diff
@@ -1170,15 +1171,17 @@ def test_writes_deletes_during_pod_down(self):
         LOGGER.info("Performed WRITEs on %s buckets for background DELETEs", del_total_bkt)
 
         LOGGER.info("Starting WRITEs on %s buckets in background", wr_bucket)
-        args = {'test_prefix': self.test_prefix, 'test_dir_path': self.test_dir_path,
+        test_write_prefix = 'test-write-26446'
+        args = {'test_prefix': test_write_prefix, 'test_dir_path': self.test_dir_path,
                 'skipget': True, 'skipdel': True, 'bkts_to_wr': wr_bucket, 'output': wr_output}
         thread1 = threading.Thread(target=self.ha_obj.put_get_delete,
                                    args=(event, s3_test_obj,), kwargs=args)
 
         LOGGER.info("Starting DELETEs of %s buckets in background", del_bucket)
         get_random_buck = self.system_random.sample(written_bck, del_bucket)
+        del_random_buck = get_random_buck.copy()
```

Comment:
Is deepcopy needed here? I guess with only copy, changes in del_random_buck will reflect in get_random_buck.

Reply:
> Is deepcopy needed here? I guess with only copy, changes in del_random_buck will reflect in get_random_buck.

Yes, since we are removing value of del_random_buck (ie bkt_list) in function, only copy will make change in main list to if change in indexed value.
![image](https://user-images.githubusercontent.com/83633380/176103640-396ae5c6-1281-4f25-a9bb-51f17fa8fe43.png)